core/src/main/java/io/aiven/klaw/controller/TopicController.java

```diff
@@ -126,6 +126,19 @@ public ResponseEntity<List<TopicRequestsResponseModel>> getTopicRequests(
         HttpStatus.OK);
   }
 
+  /**
+   * @param topicReqId requestId of topic
+   * @return Topic Request details
+   */
+  @RequestMapping(
+      value = "/getTopicRequest",
```

Comment:
I think for this request mapping the convention should be GET /topic/request/{topicRequestId} or 
GET /request/topic/{topicRequestId}